### PR TITLE
[CONSOLE] Use translated resource rather than hardcoded text

### PR DESCRIPTION
**Pull request by KRosUser**
## Purpose

- Current implementation use hardcoded text for "Default" title and does not use the translated ressource to be used as CPL name

## Proposed changes

- Load the localized CPL name as title, similarly to [4631](https://github.com/KRosUser/reactos/issues/4631)

Post change :
image

### CODE REVIEW

File: dll/cpl/console/console.c [hunk 1]
```diff
@@ -237,10 +237,12 @@ InitApplet(HANDLE hSectionOrWnd)
     psh.dwFlags = PSH_PROPSHEETPAGE | PSH_PROPTITLE | /* PSH_USEHICON | */ PSH_USEICONID | PSH_NOAPPLYNOW | PSH_USECALLBACK;
 
     if (ConInfo->ConsoleTitle[0] != UNICODE_NULL)
+    {
         StringCchPrintfW(szTitle, ARRAYSIZE(szTitle), L"\"%s\"", ConInfo->ConsoleTitle);
+        psh.pszCaption = szTitle;
+    }
     else
```
Comment (SergeGautherie) [inline dll/cpl/console/console.c:244]: Add braces.

Comment (KRosUser) [reply dll/cpl/console/console.c:244]: Done

File: dll/cpl/console/console.c [hunk 2]
```diff
@@ -237,10 +237,14 @@ InitApplet(HANDLE hSectionOrWnd)
...
-        StringCchCopyW(szTitle, ARRAYSIZE(szTitle), L"ReactOS Console");
```
Comment (SergeGautherie) [inline dll/cpl/console/console.c:242]: Note, to be explicit:
en-US has 'Console' only.

**Review by SergeGautherie — APPROVED** [review]
LGTM: this matches other CPLs.

File: dll/cpl/console/console.c (hunk 2 above)
Comment (KRosUser) [reply dll/cpl/console/console.c:242]: Good point; I assume it's fine as such (i have no strong objection on this anyway)

**Review by HBelusca — APPROVED** [review]

**Pull request merged**